plugins/autopeering/autopeering.go

```diff
@@ -92,7 +91,7 @@ func createPeerDisc() {
 	}
 	log.Debugf("Master peers: %v", masterPeers)
 
-	peerDisc = discover.New(local.GetInstance(), ProtocolVersion, NetworkVersion,
+	peerDisc = discover.New(local.GetInstance(), ProtocolVersion, config.Node().GetUint32(CfgNetworkVersion),
```

Comment:
I think it would be cleaner to define a variable `autopeering.NetworkVersion` (or even function) then you don't need to read from the config here.